helm/values.yaml

```diff
@@ -165,7 +163,7 @@ resources:
 route:
   enabled: false
   path: /
-  host: 
+  host:
```

Comment:
Please run `helm template` cmd to check if its valid since `host` has no default value (' ')